### ISSUE TITLE
Frame Type always "Application" (AMD Fluid Motion Frames)

### ISSUE THREAD

**Issue by DevTechProfile** [GameTechDev/PresentMon#381]
I've integrated latest PM v2.3 into CapFrameX. Even when enabling AMD Fluid Motion Frames, Frame Type always being set to "Application". 

Did I miss something? Do I need to pass more options/arguments to the console app?! 

`Cyberpunk2077.exe,21992,0x1B4CD825FE0,DXGI,0,512,0,Composed: Flip,Application,11290392.7197,5.7686,3.3141,2.4545,3.4651,5.6450,5.6450,0.0000,NA,NA,NA,NA`

**Comment by markgalvan-intel** [comment]
Interesting. Did you specify "--track_frame_type" as an argument to the console application?

We just ran the released console application with CP2077 and AMD Fluid Motion Frames enabled and here is a snippet for the captured CSV:

Cyberpunk2077.exe,7440,0x23952EE0E10,DXGI,0,512,0,Composed: Flip,**AMD AFMF**,884162.7183,0.0000,0.0000,0.0000,0.0000,0.0000,0.0000,0.0000,28.4829,8.3031,0.0000,0.0000,NA,NA
Cyberpunk2077.exe,7440,0x23952EE0E10,DXGI,0,512,0,Composed: Flip,Application,884162.7183,14.7700,3.8962,10.8738,5.4893,13.1780,13.1780,0.0000,36.7860,7.8551,-6.9301,884162.7183,NA,NA
Cyberpunk2077.exe,7440,0x23952EE0E10,DXGI,0,512,0,Composed: Flip,**AMD AFMF**,884177.4883,0.0000,0.0000,0.0000,0.0000,0.0000,0.0000,0.0000,29.8711,7.8572,0.0000,0.0000,NA,NA

If the above doesn't help would it be possible to capture and ETL file for us to debug?

**Comment by DevTechProfile** [comment]
> Interesting. Did you specify "--track_frame_type" as an argument to the console application?

Yes, I am passing "track_frame_type" in the argument list. What's interesting, it works with XeSS FG. I've tested it on another system. AMD Fluid Motion Frames still isn't tracked correctly by PresentMon 2.3.0. The Adrenalin driver reports  twice the FPS.

**Comment by markgalvan-intel** [comment]
"The Adrenalin driver reports twice the FPS" - So the AMD driver FPS overlay is reporting double the FPS that PresentMon is reporting for Displayed FPS?

Can you please provide an ETL file for us to debug. If not, can you provide your system setup and application used for testing so we can reproduce.

**Comment by aretha-dev** [comment]
Nice to meet you.
I'm having the same thing happen to me.
I can't get the AFMF frame rate with Presentmon.
The frame type always shows as application.
I'm using the version installed with PresentMon-v2.3.0.msi.
My environment is a Ryzen 7 8700G CPU.
The GPU is an iGPU radeon780M.
The motherboard is a B650M-HDV/M.2.
The OS is Windows 11.
The games I played were ARMORED CORE VI and the demo version of Dynasty Warriors ORIGIN.
n both cases, PresentMon displayed the FPS before AFMF was applied.
The original AMD Adrenaline displayed the FPS increased by AFMF.

I don't know much about C or C++,
but is there anything I can help you with?

**Comment by markgalvan-intel** [comment]
Thank you for the system configuration information. We will test locally and debug what's happening.

**Comment by DevTechProfile** [comment]
> If not, can you provide your system setup and application used for testing so we can reproduce.

OS: Windows 11 24H2 Build 26100.2894
CPU: Ryzen 7 9800X3D
Motherboard: ASUS Rog Strix X670E-E Gaming WiFi
BIOS: 2604
GPU: RX 7900 XTX
Driver: 24.12.1
Game: Cyberpunk 2077 + latest patch


> "The Adrenalin driver reports twice the FPS" - So the AMD driver FPS overlay is reporting double the FPS that PresentMon is reporting for Displayed FPS?

Yes, that's the case.

**Comment by markgalvan-intel** [comment]
Thanks for the update. We have reproduced the issue and are currently working with AMD to resolve the issue.
